main.go

```
@@ -23,6 +23,10 @@ type apiConfig struct {
 
 //go:embed static/*
 var staticFiles embed.FS
+func unused() {
```

golangci-lint (2.13.2)

[error] func unused is unused

(unused)